public/app/components/codeEditor/codeEditor.js

```diff
@@ -73,43 +76,18 @@ angular.module('kibibitCodeEditor')
 
     vm.updateFileContent = function(filePath) {
       if (filePath !== '') {
-        FileService.getFile(filePath, function(fileContent) {
-          vm.code = fileContent.data;
+        FileService.getFile(filePath, function(fileInfo) {
+          vm.fileInfo = fileInfo.data;
+          vm.code = vm.fileInfo.content;
         });
       }
     };
-  }
-])
-
-.directive('kbChangeAceScroll', function() {
```

Comment:
It wasn't attached anywhere. It was a leftover from an experiment I did in a previous PR